### PR TITLE
Fix/storybook build

### PR DESCRIPTION
**Pull request by osmaczko**
CMake's GLOB is capturing source files from `build/*` resulting in
multiple `main` definitions.

Generally, usage of GLOB is discouraged.

Linking error:
```
[build] /usr/bin/ld: CMakeFiles/Storybook.dir/main.cpp.o: in function `main':

[build] main.cpp:(.text.startup+0x0): multiple definition of `main'; CMakeFiles/Storybook.dir/CMakeFiles/3.22.5/CompilerIdCXX/CMakeCXXCompilerId.cpp.o:CMakeCXXCompilerId.cpp:(.text.startup+0x0): first defined here

[build] collect2: error: ld returned 1 exit status
```

### CODE REVIEW

**Review by caybro — APPROVED** [review]

File: storybook/CMakeLists.txt [hunk 1]
```diff
@@ -17,18 +17,11 @@ find_package(
...
 file(GLOB_RECURSE QML_FILES "stubs/*.qml" "Storybook/*.qml" "../ui/StatusQ/*.qml" "../ui/app/*.qml")
 file(GLOB_RECURSE JS_FILES "../ui/StatusQ/*.js" "../ui/app/*.js")
-file(GLOB_RECURSE HEADERS *.h)
-if(APPLE)
-  file(GLOB_RECURSE SOURCES *.cpp *.mm)
-else()
-  file(GLOB_RECURSE SOURCES *.cpp)
-endif()
 
 add_executable(
```
Comment (endulab) [inline storybook/CMakeLists.txt:21]: @osmaczko can you add also this ? W/o this we can't build for Mac:
```
if (APPLE)
    set(MACOS_VERSION_MIN_FLAGS -mmacosx-version-min=10.14)
    set(CMAKE_OSX_ARCHITECTURES "x86_64")
endif()
```
^ @noeliaSD

**Review by endulab — CHANGES_REQUESTED** [review]

File: storybook/CMakeLists.txt (hunk 1 above)
Comment (noeliaSD) [reply storybook/CMakeLists.txt:21]: > @osmaczko can you add also this ? W/o this we can't build for Mac:
> 
> ```
> if (APPLE)
>     set(MACOS_VERSION_MIN_FLAGS -mmacosx-version-min=10.14)
>     set(CMAKE_OSX_ARCHITECTURES "x86_64")
> endif()
> ```
> 
> ^ @noeliaSD

Tested on mac with this addition!

Comment (caybro) [reply storybook/CMakeLists.txt:21]: Guys, this is git, you can add/push the change to the branch yourselves ;)

Comment (osmaczko) [reply storybook/CMakeLists.txt:26]: Added, but this change request was strange :sweat_smile: I can't test it, because I am not on mac.

Comment (caybro) [reply storybook/CMakeLists.txt:26]: Exactly what I'd suggested above :D

Comment (noeliaSD) [reply storybook/CMakeLists.txt:26]: You should trust us.. ;)

File: storybook/main.cpp [hunk 2]
```diff
@@ -8,9 +8,9 @@ int main(int argc, char *argv[])
     QCoreApplication::setAttribute(Qt::AA_EnableHighDpiScaling);
 #endif
     QGuiApplication app(argc, argv);
-    app.setOrganizationName("Status");
-    app.setOrganizationDomain("status.im");
-    app.setApplicationName("Status Desktop Storybook");
+    QGuiApplication::setOrganizationName("Status");
```
Comment (micieslak) [inline storybook/main.cpp:11]: Is it better then the previous version? I'm just curios :)

**Review by noeliaSD — APPROVED** [review]

File: storybook/CMakeLists.txt [hunk 3]
```diff
@@ -10,25 +10,23 @@ set(CMAKE_AUTORCC ON)
...
+endif()
+
 find_package(
   Qt5
   COMPONENTS Core Quick QuickControls2
   REQUIRED)
 
 file(GLOB_RECURSE QML_FILES "stubs/*.qml" "Storybook/*.qml" "../ui/StatusQ/*.qml" "../ui/app/*.qml")
 file(GLOB_RECURSE JS_FILES "../ui/StatusQ/*.js" "../ui/app/*.js")
-file(GLOB_RECURSE HEADERS *.h)
```
Comment (micieslak) [inline storybook/CMakeLists.txt:20]: Ouch, that was really bad (copy pasted from StatusQ, the same problem there).

**Review by micieslak — APPROVED** [review]
LGTM! Thank you! One small question.

File: storybook/main.cpp (hunk 2 above)
Comment (osmaczko) [reply storybook/main.cpp:11]: This is what clang-tidy suggested to me: https://clang.llvm.org/extra/clang-tidy/checks/readability/static-accessed-through-instance.html

**Review by endulab — APPROVED** [review]

**Pull request merged**